tools/test-proxy/Azure.Sdk.Tools.TestProxy/Matchers/HeaderlessMatcher.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using System.Text;
+using Azure.Sdk.Tools.TestProxy.Common;
+
+namespace Azure.Sdk.Tools.TestProxy.Matchers
+{
+    public class HeaderlessMatcher : RecordMatcher
+    {
+        /// <summary>
+        /// This matcher adjusts the "match" operation to ignore header differences when matching a request. Be aware that wholly ignoring headers during matching might incur unexpected issues down the line.
+        /// </summary>
+        public HeaderlessMatcher() : base(true)
```

Comment:
Is there a way to ignore just specific headers?

Reply:
Those ideas came up at https://github.com/azure/azure-sdk-tools/issues/2303 after this PR was merged 😛